### ISSUE TITLE
[Docs?]: Add more examples in theme customization for other uses of `pando-light`/`pando-dark` classes

### ISSUE THREAD

**Issue by saurookadook** [pluralsight/pando#1847]
### Latest version

- [X] I have checked the latest version

### Summary 💡

In the [Customizing with CSS](https://design.pluralsight.com/docs/reference/general/themes/customizing/#customizing-with-css) section, it's not entirely clear that the `pando-light`/`pando-dark` classes can be used to change token colors on an individual basis.

My initial thought is to add another example to the end of that section, kinda like this:

> Additionally, you can utilize the `pando-light`/`pand-dark` classes to selectively use colors from those themes for a given token.
>
> For example, if you're using the dark theme colors but want to use the color for a token from the light theme palette, including the `pando-light` class in the selector will accomplish that:
> ```css
> .some-custom-component {
>     color: var(--ps-text); /* resolves to #D1D2E6 */
> }
>
> .some-custom-component.pando-light {
>     color: var(--ps-text); /* resolves to #3E5674 */
> }
> 
> ```
>
> In opposition, you can use the `pando-dark` class if you're otherwise using light theme colors:
> ```css
> .some-custom-component {
>     color: var(--ps-text); /* resolves to #3E5674 */
> }
> 
> .some-custom-component.pando-dark {
>     color: var(--ps-text); /* resolves to #D1D2E6 */
> }
> ```
> 

However, I wonder if it would be better to add a section that explicitly calls out that the `pando-dark` and `pando-light` classes are provided out of the box, presenting them as another tool for customization. With the current examples, it's not entirely clear that, after overriding the color token, you wouldn't still have to add the `pando-light` class yourself when changing themes.

In other words, I think it could **_much_** more clear how themes are applied out of the box in Pando. 🙂
